app.rb

```diff
@@ -0,0 +1,76 @@
+require 'sinatra'
+require_relative 'lib/database_connection'
+require_relative './lib/user_repository'
+require_relative './lib/peep_repository'
+require_relative './lib/peep'
+require_relative './lib/user'
+require 'rack/test'
+
+DatabaseConnection.connect('chitter_database_test')
+
+class Application < Sinatra::Base
+
+  def set_all_peeps
+    peep_repo = PeepRepository.new
+    user_repo = UserRepository.new
+    peeps = peep_repo.all.reverse!
```

Comment:
Really interesting to see different approaches used to achieve same results.(to get peeps displayed in reverse chronological order)